src/map/clif.c

```diff
@@ -17107,6 +17110,7 @@ void clif_snap( struct block_list *bl, short x, short y ) {
 	clif->send(buf,packet_len(0x8d2),bl,AREA);
 }
 
+#ifdef ENABLE_MONSTER_HP_BAR
```

Comment:
The function must exist at all times, so the ifdef should go inside, rather than outside